katana/wui/core/static/core/js/main.js

```diff
@@ -1242,4 +1242,67 @@ var katana = {
 
   },
 
+    utils: {
+
+        getRelativeFilepath: function (basePath, path) {
+          /* This function gets relative path (out of a given path argument) from a basePath and path */
+            if (basePath.indexOf('\\') > -1) {
+                basePath = basePath.replace('\\', '/');
+            }
+            if (path.indexOf('\\') > -1) {
+                path = path.replace('\\', '/');
+            }
+            var basePathSeries = basePath.split('/');
+            var pathSeries = path.split('/');
+            var hold = 0;
+            for (var i=0; i<basePathSeries.length && i < pathSeries.length; i++) {
+                if (basePathSeries[i] !== pathSeries[i]){
+                    hold = i;
+                    break;
+                }
+            }
+            var output = "";
+            for (i=(basePathSeries.length-1); i > hold; i--) {
+                output += "../"
+            }
+            if (output !== "") {
+                output = output.slice(0, -1);
+            }
+            for (i=hold; i<pathSeries.length; i++) {
+                output += "/" + pathSeries[i]
+            }
+            if (output.startsWith("/")) {
+                output = output.slice(1, output.length);
+            }
+            return output
+        },
+
+        getAbsoluteFilepath: function (basePath, relativePath) {
```

Comment:
Similar situation as the previous comment
`
katana.utils.getAbsoluteFilepath("/home/ka/Desktop/cases/", "warriorframework/katana");
"/home/ka/Desktop/cases/warriorframework/katana"`
`katana.utils.getAbsoluteFilepath("/home/ka/Desktop/cases", "warriorframework/katana");
"/home/ka/Desktop/warriorframework/katana"
`
The second example should append a / at the end if it doesn't have one

Reply:
Addressed.